src/data/sign.lean

```diff
@@ -0,0 +1,44 @@
+/-
+Copyright (c) 2022 Eric Rodriguez. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Rodriguez
+-/
+import order.basic
+
+/-!
+# Sign function
+
+This file defines the sign function for types with zero and a decidable less-than relation, and
+proves some basic theorems about it.
+-/
+
+
+variables {α : Type*} [has_zero α]
+
+/-- The sign of an element - 1 if it's positive, -1 if negative, 0 otherwise. -/
+def sign [has_lt α] [decidable_rel ((<) : α → α → Prop)] (a : α) :=
```

Comment:
Actually, should we make this an `order_hom`?

Reply:
sure, that seems sensible!

Reply:
is it on purpose that the `order_hom` simps projection is `coe`? shouldn't it be `apply`?